.npmignore

```diff
@@ -4,3 +4,7 @@ npm-debug.log
 coverage
 .gitignore
 .DS_Store
+.npmignore
+__tests__
```

Comment:
Im skeptical anyone ever does that given that nearly all modules exclude tests. Doing a google search, nearly all recommendations are to not publish tests and i cant find any examples of people complaining because tests of a dependency dont run.. the npm docs themselves dont give any recommendations i can see either way though.